packages/@o3r/core/schematics/application/index.ts

```diff
@@ -0,0 +1,58 @@
+import { apply, chain, externalSchematic, MergeStrategy, mergeWith, move, renameTemplateFiles, Rule, schematic, strings, template, url } from '@angular-devkit/schematics';
+import { getPackagesBaseRootFolder, getWorkspaceConfig, isNxContext } from '@o3r/schematics';
+import * as path from 'node:path';
+import type { NgGenerateApplicationSchema } from './schema';
+import type { PackageJson } from 'type-fest';
+
+/**
+ * Add an Otter application to a monorepo
+ *
+ * @param options Schematic options
+ */
+export function generateApplication(options: NgGenerateApplicationSchema): Rule {
+  const cleanName = strings.dasherize(options.name);
+
+  const addProjectSpecificFiles = (targetPath: string, rootDependencies: Record<string, string | undefined>) => {
+
+    return mergeWith(apply(url('./templates'), [
+      template({
+        ...options,
+        name: cleanName,
+        rootDependencies
+      }),
+      move(targetPath),
+      renameTemplateFiles()
+    ]), MergeStrategy.Overwrite);
+  };
+
+  const ngCliUpdate: Rule = (tree, context) => {
+    const config = getWorkspaceConfig(tree);
+    if (!config) {
+      throw new Error('No workspace configuration file found');
+    }
+    if (isNxContext(tree)) {
+      throw new Error('NX is not supported.');
+    }
+
+    const defaultRoot = getPackagesBaseRootFolder(tree, context, config, 'application');
+
+    /** Path to the folder where to generate the new application */
+    const targetPath = path.posix.resolve('/', options.path || defaultRoot, cleanName);
```

Comment:
on Windows, this will resolve as the actual absolute path (like `c:\...`) which won't work on the Tree
it would be nice to add some unit tests (even if it's not very stable on windows at the moment) to make sure it works as expected

Reply:
```suggestion
    const targetPath = path.posix.join('/', options.path || defaultRoot, cleanName);
```